Sql Server Integration Tutorial is Missing Razor Imports

### Describe the issue or suggestion

The Sql Server Integration Tutorial is Missing Razor Imports, so when you work through it and build you get the error below.  The solution is to a `@using Microsoft.EntityFrameworkCore` to the `_Imports.razor` file - if you're interested in having the tutorial updated to include this step so that it works as an end to end tutorial let me know and I'll put in a PR.

`1>C:\<redacted>\Tutorials\AspireSQLEFCore\AspireSQLEFCore\Components\Pages\Home.razor(49,41,49,52): error CS1061: 'DbSet<SupportTicket>' does not contain a definition for 'ToListAsync' and no accessible extension method 'ToListAsync' accepting a first argument of type 'DbSet<SupportTicket>' could be found (are you missing a using directive or an assembly reference?)
1>C:\<redacted>\Tutorials\AspireSQLEFCore\AspireSQLEFCore\Components\Pages\Home.razor(58,41,58,52): error CS1061: 'DbSet<SupportTicket>' does not contain a definition for 'ToListAsync' and no accessible extension method 'ToListAsync' accepting a first argument of type 'DbSet<SupportTicket>' could be found (are you missing a using directive or an assembly reference?)
1>Done building project "AspireSQLEFCore.csproj" -- FAILED.`

https://learn.microsoft.com/en-us/dotnet/aspire/database/sql-server-integrations

--- a/docs/database/snippets/tutorial/aspiresqlefcore/AspireSQLEFCore/Components/Pages/Home.razor
+++ b/docs/database/snippets/tutorial/aspiresqlefcore/AspireSQLEFCore/Components/Pages/Home.razor
@@ -1,5 +1,6 @@
 ﻿@page "/"
 @inject TicketContext context
+@using Microsoft.EntityFrameworkCore
 
 <div class="row">
     <div class="col-md-6">

